keyboards/lw75/readme.md

```diff
@@ -0,0 +1,22 @@
+# LW-75
+
+![LW67](https://i.imgur.com/mcM1TTi.jpg)
```

Comment:
```suggestion
![LW67](https://i.imgur.com/mcM1TTih.jpg)
```

Reply:
Rejected due to incorrect original link. New change includes your suggested change.